packages/internal-scripts/package.json

```diff
@@ -15,9 +15,26 @@
     "treemirror": "bin/treemirror.js"
   },
   "scripts": {
-    "build": "npm run clean && npm run build:ts",
-    "build:ts": "tsc --build",
-    "clean": "rm -rf lib/"
+    "build": "wireit",
+    "build:ts": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:ts"
+      ]
+    },
```

Comment:
Would this do anything different than just running `build:ts`? And is `npm run clean` no longer necessary since `"if-file-delete"` should cover all cases we'd actually need to clean, since otherwise outputs would get overwritten?

Reply:
It's the same as running build:ts directly, yes

And yes, `if-file-delete` combined with `tsc --build` should behave the same as always running clean before building